tests/common-pg.cpp

```diff
@@ -129,7 +129,7 @@ const std::string &tempdb::conninfo() const {
     return m_conninfo;
 }
 
-void tempdb::setup_extension(conn_ptr db, const std::string &extension, ...) {
+void tempdb::setup_extension(conn_ptr db, const std::string extension, ...) {
```

Comment:
I'm not sure of a better way to handle this, but `va_start(ap, extension);` isn't valid with a reference type.